contact.html

```diff
@@ -26,6 +26,7 @@
         <!--- Main Body /> -->
         <div class="contactUs">
             <div class="contactUsForm">
+        <!-- Note -->
```

Comment:
Instead of using the `onsubmit` property, it'll work better if you use `addEventListener` in your Javascript.  By that I mean, at the very end of you contactUs.js file, you would do something like `document.getElementById('myform').addEventListener('submit', validate)`. Using an event listener means that your validate function will receive an `Event` object that it can use to stop the form from being submitted.  Here's an example of that: https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault#examples

Rewriting your code to use an event listener though would require changing how you're defining the `validate` function to something like `let validate = function(event) { ... }`